### ISSUE TITLE
Unable to create account in applause.expensifail.com domain

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#41362]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.68-3
**Reproducible in staging?:** y
**Reproducible in production?:** no
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:**

## Action Performed:
1. Go staging.new.expensify.com
2. Enter any email address which does not have expensify account in the domain applause.expensifail.com 
3. Click continue
## Expected Result:
Taken to entering magic code page
## Actual Result:
Error message appears ""Cannot get account details, please try again or contact [concierge@expensify.com"]

NOTE: No issue with public domain gmail.com and sign in with existing applause.expensifail.com domain accounts
## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/38435837/ec5ec596-d3eb-4a09-aea9-c3f9ef7d2f96

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by francoisl** [comment]
This may have been a temporary backend issue and seems to be working fine now, can you try again please?

<img width="1267" alt="image" src="https://github.com/Expensify/App/assets/2229301/85f64aa8-a937-44fe-8dd2-7236046ef73e">

If you're still running into an issue, please make sure to paste the email address you were using in comment too.

**Comment by francoisl** [comment]
Oh I see, you were using the domain `applause.expensifail.com`, not `applause.expensify.com`. Looks like it might be a backend issue, looking now.

**Comment by francoisl** [comment]
Yeah this is a backend issue, it's happening on OldDot as well. It only seems to be affecting `applause.expansifail.com` from what I can tell. Going to remove the blocker label for NewDot.

<img width="328" alt="image" src="https://github.com/Expensify/App/assets/2229301/6f25ccfb-b1e2-435e-98d0-a97ab0939905">

**Comment by kavimuru** [comment]
@francoisl just to let you know,
The accounts on applause.expensifail.com are currently in beta. We run many NewDot regression test cases using these domain accounts since testers have HT account in this domain. Additionally, we're unable to run a test case in OldDot.

**Comment by francoisl** [comment]
I identified the issue, working on a fix now.

**Comment by francoisl** [comment]
Not 100% fixed yet sorry, one sec. I'll let you know when it's ready to go.

**Comment by francoisl** [comment]
Ok should be fixed for real now. 

<img width="658" alt="image" src="https://github.com/Expensify/App/assets/2229301/6f04a83c-f80e-492f-ae5d-a18f00b437c9">
